packages/shared/src/graphql/actions.ts

```diff
@@ -2,6 +2,14 @@ import request, { gql } from 'graphql-request';
 import { graphqlUrl } from '../lib/config';
 
 export enum ActionType {
+  CreateSquad = 'create_squad',
+  JoinSquad = 'join_squad',
+  EditWelcomePost = 'edit_welcome_post',
+  SquadFirstComment = 'squad_first_comment',
+  SquadFirstPost = 'squad_first_post',
+  MyFeed = 'my_feed',
+  SquadInvite = 'squad_invite',
+  BrowserExtension = 'browser_extension',
   Notification = 'notification',
```

Comment:
This one is different though, make sure it's aligned with the DR

Reply:
It is already used as this in the PR that @sshanzel merged. @sshanzel maybe you can look if this should be `enable_notification`, I might have missed something.

Reply:
Right. That is the placeholder value. We should have it converted to `enable_notification`. We haven't released this to prod (app.daily.dev) so it is safe for us to update here.